etp-backend/src/main/clj/solita/etp/api/valvonta_oikeellisuus.clj

```diff
@@ -67,7 +69,9 @@
                      :responses  {200 {:body oikeellisuus-schema/Valvonta}}
                      :access     (some-fn rooli-service/paakayttaja? rooli-service/laatija?)
                      :handler    (fn [{{{:keys [id]} :path} :parameters :keys [db whoami]}]
-                                   (r/response (valvonta-service/find-valvonta db id)))}
+                                   (api-response/get-response
+                                     (valvonta-service/find-valvonta db id)
+                                     (str "Energiatodistus " id " does not exists.")))}
```

Comment:
Pitäisikö tässä olle Valvonta <id> does not exist? Vai onko valvonnan ja ET:n id sama aina?

Reply:
joo valvonnan ja et:n id:t on aina sama

Reply:
nythän se seuraa siitä että ne on fyysisesti samassa taulussa, mutta vaikka ne olisi eri tauluissa niin siitä huolimatta niissä kannattaa käyttää samaa id:tä

Reply:
se että ne on samassa taulussa on vain toteutusyksityiskohta jonka ei pitäisi näkyä ulos rajapinnasta

Reply:
Tässä on vielä se että et:llä on aina valvontatiedot. Eli tämä tosiaan palauttaa 404 vain jos et:tä ei ole olemassa